types/aggregator.go

```diff
@@ -2,11 +2,16 @@ package types
 
 import "time"
 
+const (
+	DefaultAggregatorVersion = "v0.1.12"
```

Comment:
I wonder somethimes if these defaults should be on this repo 🤔

Reply:
@nicolasparada me too.